game/utils/parser.cpp

```diff
@@ -2,16 +2,16 @@
 
 using namespace Tempest;
 
-Vec2 Parser::loadVec2(const Daedalus::ZString& src) {
-  return loadVec2(std::string_view(src.c_str()));
-  }
-
 Vec2 Parser::loadVec2(std::string_view src) {
   if(src=="=")
     return Vec2();
 
-  float       v[2] = {};
-  const char* str  = src.data();
+  float v[2] = {};
+  char  buf[256];
+  std::memcpy(buf, src.data(), src.length() > 254 ? 254 : src.length());
+  buf[src.length()] = 0;
```

Comment:
overflow?

Reply:
I tried to account for that but missed the most obvious issue :D